src/app/page.tsx

```diff
@@ -1,113 +1,63 @@
-import Image from 'next/image'
+"use client"
+import { Alert, Container, Snackbar } from '@mui/material'
+
+import TOdoList from '../../components/TodoList'
+import TOdoForm from '../../components/TodoForm'
```

Comment:
Ensure to use the correct naming convention on these files:

`TodoList` and `TodoForm`